layers/meta-tegrademo/scripts/deploy

```diff
@@ -0,0 +1,78 @@
+#! /bin/bash
```

Comment:
**Minor:**
Could you please use `#!/bin/sh` instead to make the script compatible with the others shell interpreters like zsh ?

Reply:
Since they're self-contained scripts, bash is probably OK.  It's unlikely that someone will not have the bash interpreter around, even if they don't use it themselves.

Reply:
Agree but I thought just to be consistent with the other scripts.

Reply:
done!